### PR TITLE
changing the account model from the blog app to account app

### PR DESCRIPTION
**Pull request by hellen-22**

### CODE REVIEW

**Review by JimmyTron — APPROVED** [review]

**Review by JimmyTron — CHANGES_REQUESTED** [review]
you should also change the registered models on the admin.py 

```
from django.contrib import admin
from django.contrib.auth.admin import UserAdmin
from .models import *

"""
Registering user model to the admin and extending the UserAdmin so that the user admin page looks like the default user admin panel
List_display specifies the attributes that will appear on the admin page.
add_fieldsets specifies the attributes to be using when creating a user on the admin panel
"""
@admin.register(User)
class CustomUserAdmin(UserAdmin):
    list_display = ('email', 'is_staff', 'is_active',)
    add_fieldsets = (
        (None, {
            'classes': ('wide',),
            'fields': ('email', 'username','password1', 'password2', 'is_staff', 'is_active')}
        ),
    )

admin.site.register(Profile)
```

**Review by JimmyTron — APPROVED** [review]
Looks fine

File: accounts/models.py [hunk 1]
```diff
@@ -11,10 +11,22 @@
 class User(AbstractUser):
     email = models.EmailField(unique=True)
```
Comment (sangkips) [inline accounts/models.py:12]: Add max_length to the email to avoid users creating long fake emails

**Review by sangkips — CHANGES_REQUESTED** [review]

**Review by sangkips — APPROVED** [review]
Good

**Pull request merged**